tests/openuri.c

```diff
@@ -478,3 +478,61 @@ test_open_directory (void)
   while (!got_info)
     g_main_context_iteration (NULL, TRUE);
 }
+
+void
+test_scheme_supported(void) {
+  g_autoptr(GError) error = NULL;
+
+    g_autoptr(GDBusProxy) proxy = g_dbus_proxy_new_for_bus_sync (
```

Comment:
Instead of using the generic `GDBusProxy` you can use the generated `XdpDbusOpenURI *xdp_dbus_open_uri_proxy_new_sync()` and related functions.